content/post/jamme-jellyfish-release-notes/index.md

```diff
@@ -625,41 +625,40 @@ Starting with 22.04 LTS, besides the standard device-specific preinstalled image
 
 Other operating systems are not displayed in the boot menu anymore, unless Ubuntu has been installed alongside another operating system. Once all other operating systems are removed from the machine, detection of other operating systems is disabled, and to re-enable if after installing another OS, you will have to delete `/boot/grub/grub.cfg` and immediately run `update-grub` again.
 
-# Known Issues
+# 알려진 이슈
 
-As is to be expected, with any release, there are some significant known bugs that users may run into with this release of Ubuntu. The ones we know about at this point (and some of the workarounds), are documented here so you don’t need to spend time reporting these bugs again:
+우분투 릴리즈에서 발생할 수 있는 몇가지 알려진 버그가 존재합니다. 이 시접에서 우리가 알고 있는 것(과 일부 해결 방법)이 문서화하였으며 여기에 적혀있는 버그에 많은 시간을 할애하지 않길 바랍니다.
+## 리눅스 커널
 
-## Linux kernel
+아직 없음.
 
-Nothing yet.
+## 보안
+* OpenSSL 3.0.2에서는 [터키 로케일](https://launchpad.net/bugs/1968997)에서는 동작하지 않습니다. 수정 사항이 제공될 때까지 이 문제를 해결하려면 `LC_CTYPE=C.UTF-8`을 설정하여 openssl을 사용하는 것이 좋습니다. (셸: `export LC_CTYPE=C.UTF-8`) 이렇게 하면 터키어 규칙에 따라 시스템 수준에서 문자열의 대문자를 재정의하지 않습니다.

```

Comment:
> 마지막 문장은 이렇게 하면 시스템이 망가지므로 시스템 레벨에서 재정의 하는것을 권장하지 않는다는 내용으로 보입니다.

해당 내용이 이해가 되지 않는데요. 셸 환경 설정을 통해서 시스템상 문제가 발생할 수 있는 부분을 수정을 통해서 피할 수 있다는 내용인데 어떻게 "시스템 레벨에서 재정의 하는것을 권장하지 않는다"라고 이해할 수 있는지 궁금합니다.

Reply:
마지막 문장이 아래 문장인데요. 저는 아래와 같은 내용으로 이해 했습니다. 대략 셸로 문제 회피가 가능하지만, 그러면 다른 문제가 또 생기니 권장하지 않는다는 것으로 이해하고 있습니다.
This will break capitalization of strings according to rules for the Turkish language, so is not recommended to be overridden at the system level.

This will break 이는 (Workaround 를 적용 하는 것은) 문제를 일으킨다.
capitalization of strings according to rules for the Turkish language, 터키어 규칙에 따른 문자열 대문자 사용에 
so is not recommended to be overridden at the system level. 그러므로 시스템 레벨에서 재정의 하는 것을 권장하지 않는다.

Reply:
뭔가 중간에 번역하면서 빠진부분을 봤나보네요 수정했습니다.